catalogues/README.md

```diff
@@ -0,0 +1,22 @@
+## Update catalogues
+Update catalogues for katconfig repo for offline planning with AstroKAT
+
+If you have access: update the catalogues to ensure they are up to date with katconfig    
+`git clone https://github.com/ska-sa/katconfig.git`
+
+Update astrokat catalogues manually
+```
+git clone https://github.com/ska-sa/astrokat.git
+cd astrokat/catalogues
+for file in Lband-*calibrators.csv ; do echo $file ; cp $file bak/ ; cp ../../katconfig/user/catalogues/$file . ; done
```

Comment:
I have unfortunately not worked with U-band observations, or had interactions with astronomers who did.
But, from what I can see is that it works pretty much like L-band observations, so I updated the script that "sync" the `astrokat` catalogues with `katconfig`, to grab all calibrator catalogues.

This will expose them to the user and if my very broad assumption fails, hopefully the users will raise issues that can be corrected.